modules/dhcp_common/dhcp_common.rb

```diff
@@ -32,4 +32,12 @@ class NotImplemented < RuntimeError; end
   def self.ipv4_to_i(ipv4_address)
     ipv4_address.split('.', 4).inject(0) { |i, octet| (i << 8) | octet.to_i }
   end
+
+  def self.i_to_ipv4(ipv4_as_i)
```

Comment:
I think it's possible to use the IPAddr module for this:
```IPAddr.new(ipv4_as_i, Socket::AF_INET).to_s```

Reply:
Probably, but again, didn't want to instantiate an IPAddr just to do the conversion.

Reply:
I don't mind any approach, but can you please add few unit tests for this method if you prefer the lower-level implementation? I know it is covered by high-level tests, but just to see regressions when we touch this code in the future.

Reply:
I'd prefer the built in method because it's less code to write and test.

Reply:
If there is a test down below, I am fine. This is being called many times down there and ruby sucks in memory management at the moment.

Reply:
there is: https://github.com/theforeman/smart-proxy/pull/543/files#diff-ea91ddc372683c1cdd90b093548992c2R12